source/agora/test/Base.d

```diff
@@ -1757,6 +1751,9 @@ public struct TestConf
 
     /// The amount of a penalty for slashed validators
     Amount slash_penalty_amount = 10_000.coins;
+
+    /// Genesis timestamp
+    ulong genesis_timestamp = 1609459200; // 2021-01-01:00:00:00 GMT
```

Comment:
IIIUC the default value is never used ? Is there a reason to make this accessible in `TestConf` ?

Reply:
Removed this.